Logstash monitoring doc improvements. (#16208)
elastic/logstash: docs/static/monitoring/monitoring-prereq-setup-es.asciidoc

//[[set-up-monitoring]]
//=== Set up {es} monitoring (stack monitoring only)

> To bind {ls} metrics to an {es} cluster, also set up {ref}/monitoring-overview.html[{es} monitoring].

Reviewer: ```suggestion
To bind {ls} metrics to an {es} cluster, set up {ref}/monitoring-overview.html[{es} monitoring].
```

> With *Stack Monitoring* it is recommended to set up {ref}/monitoring-overview.html[{es} monitoring] together. Otherwise, {ls} becomes a Standalone Cluster.

Author: review note: not sure if it is okay to mention standalone cluster but if user doesn't configure the ES monitoring, LS becomes an orphan.

> With *Stack Monitoring* it is recommended to set up {ref}/monitoring-overview.html[{es} monitoring] together. Otherwise, {ls} becomes a Standalone Cluster.

Reviewer 2: ```suggestion
To bind {ls} metrics to an {es} cluster, also set up {ref}/monitoring-overview.html[{es} monitoring].
```

Author: Agreed, I have seen multiple standalone issues under KB repo and better not to mention for now.
